README.md

```diff
@@ -1 +1,38 @@
-# task
\ No newline at end of file
+# Task
+
+Task is a [CLI](https://en.wikipedia.org/wiki/Command-line_interface) that automates the software development process.
+
+## Problem Statement
+
+The software development process consists of several interactions with different softwares: reading an issue on Jira, creating a PR on Github, and so on.
```

Comment:
The software development process consist of several interactions with different softwares. It would be a saving of time if a program could make the bridge between you and a software, updating your tasks status automatically, creating a pull request for you each time you have to work on something and so on.